apps/home-dashboard/package.json

```diff
@@ -2,31 +2,32 @@
   "name": "home-dashboard",
   "private": true,
   "version": "0.0.0",
-  "type": "module",
+  "type": "commonjs",
```

Comment:
no idea what it's this for... but it fix an error that ESLint was showing after I renamed the cjs files to js